### ISSUE TITLE
Breadcrumb HOOK disables 404 Page / Status Code = 200, for pages not existing

### ISSUE THREAD

**Issue by joeherold** [alnv/catalog-manager#167]
https://github.com/alnv/catalog-manager/blob/5ea4e2ad7a40af8a43ef698ec7077c243d6679a9/library/alnv/CatalogBreadcrumb.php#L8

https://github.com/alnv/catalog-manager/blob/5ea4e2ad7a40af8a43ef698ec7077c243d6679a9/library/alnv/CatalogBreadcrumb.php#L11

Your registered hook in 
```php
// config.php LINE 93
$GLOBALS['TL_HOOKS']['generateBreadcrumb'][] = [ 'CatalogManager\CatalogBreadcrumb', 'initialize' ];
```
messes up the 404 page. Contao thinks ( i guess it is the usage of ```php \Input::get('auto_item')``` that the route exists, but it doesn. So 404 is not rendered. and return code is 200... not good

**Comment by joeherold** [comment]
Had this issue in Contao 4.4 and 4.6. Actually commented out your HOOK

**Comment by alnv** [comment]
Should be fixed in v1.27.2

**Comment by alnv** [comment]
Bugfix is available now in v1.27.2
